pkg/web/router/router.go

```diff
@@ -70,38 +70,65 @@ func SetAppRoutes(r *echo.Group, appsHandler apps.HTTPHandler) {
 	//     description: App not found
 	r.GET("/apps/{id}", appsHandler.GetAppByID) // TODO: Implement correctly the call of the method passing the parameters
 
-	// swagger:operation PUT /apps/{id} App
+	// swagger:operation PUT /apps/:id/versions Version
 	//
-	// Update a single app using the app id, including updating version information
+	// Update all versions informed of an app using the app id, including updating version information
 	// ---
-	// summary: Update app by id
+	// summary: Update 1 or more versions of an app
+	// operationId: UpdateAppVersions
+	// produces:
+	// - application/json
+	// parameters:
+	// - name: id
+	//   in: path
+	//   description: The id for the app that will have its versions updated
+	//   required: true
+	//   type: string
+	// - name: body
+	//   in: body
+	//   description: Updated 1 or more versions of an app
+	//   required: true
+	//   schema:
+	//     $ref: '#/definitions/Version'
+	// responses:
+	//   200:
+	//     description: successful update
+	//   400:
+	//     description: Invalid app and/or versions supplied
+	//   404:
+	//     description: App not found
+	r.PUT("/apps/:id/versions", appsHandler.UpdateAppVersions)
+
+	// swagger:operation PUT /apps/:id/versions VersionRes
+	//
+	// Disable/Enable all versions of an app
+	// ---
+	// summary: Update all versions of an app
 	// operationId: updateApp
 	// produces:
 	// - application/json
 	// parameters:
 	// - name: id
 	//   in: path
-	//   description: The id for the app that needs to be updated.
+	//   description: The id for the app that will have all its versions updated
 	//   required: true
 	//   type: string
 	// - name: body
 	//   in: body
-	//   description: Updated app object
+	//   description:
 	//   required: true
 	//   schema:
-	//     $ref: '#/definitions/App'
+	//     $ref: '#/definitions/VersionRes'
 	// responses:
 	//   200:
-	//     description: successful operation
-	//     schema:
-	//       $ref: '#/definitions/App'
+	//     description: successful update
 	//   400:
 	//     description: Invalid app supplied
 	//   404:
 	//     description: App not found
-	r.PUT("/apps/{id}", appsHandler.UpdateApp) // TODO: Implement correctly the call of the method passing the parameters
```

Comment:
Since this type of endpoint is non standard REST but will make the logic easier. My proposal is

Endpoint: `/apps/:id/versions/batch-disable`
Method: `POST`
Request Body: { disableMessage: "" }

Reply:
Is not better we use the boolean? Shows not reasonable the need to enable all as well?

Reply:
your current implementation does not show a boolean on this endpoint.